jac-uk/admin
# Feature/apply 1209 Update character section of the application form

## What's included?

[apply: Update character section of the application form#1209](https://app.zenhub.com/workspaces/platform-development-5ea838cd2aec471eb6d14139/issues/gh/jac-uk/apply/1209)

- Update the character guidance options.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
[Example exercise](https://jac-admin-develop--pr2574-feature-apply-1209-u-9x23o2ed.web.app/exercise/Biyjd07Xz2usL9yXjtjV/reports/character-issues)

1. Go to Character Annex Report of an exercise.
2. Check if the character guidance options are updated.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context

Demo:

https://github.com/user-attachments/assets/527753b0-57dc-44fd-9c7d-0dfa3f4dd0df

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
